docs/design/windows-design.md

```diff
@@ -20,9 +20,10 @@ forwarded correctly.
 
 <img src="../assets/hns_integration.svg" width="600" alt="HNS Integration">
 
-SNAT based on OpenFlow is needed to make sure the containers can access the external address.
-The SNATed address is using the IP configured on the OVS bridge. Some additional OpenFlow entries
-are installed to assist in identifying and forwarding the external traffic.
+Windows NetNat is configured to make sure the containers can access the external address. The packet
```

Comment:
make sure the containers can access external addresses

Reply:
Probably "containers" -> "Pods".

Reply:
updated.